api/server/httputils/write_log_stream.go

```diff
@@ -85,6 +86,7 @@ func (s byKey) Swap(i, j int) {
 func stringAttrs(a backend.LogAttributes) string {
 	var ss byKey
 	for k, v := range a {
+		k, v := url.QueryEscape(k), url.QueryEscape(v)
```

Comment:
Why URL escaping? It looks like this is added to the log output stream, not a URL.